### DISCUSSION TITLE
Display Names

### DISCUSSION THREAD

**Discussion by L-O-G-A-N** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#245] [Q&A] [answered]
How would I go about checking display names without actually changing my name to the specific name?

**Comment by Jaren8r** [comment, marked as answer]
You can just use the displayName endpoint - https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/AccountService/Account/Lookup/Displayname.md

**Reply by L-O-G-A-N** [reply]
thanks for the response, can you please give me a code snippet, python preferably of how exactly I would use the endpoint? Thanks
